src/uwtools/cli.py

```diff
@@ -693,9 +705,12 @@ def _check_file_vs_format(file_arg: str, format_arg: str, args: Args) -> Args:
     return args
 
 
-def _check_quiet_vs_verbose(args) -> Args:
-    if args.get(STR.quiet) and args.get(STR.verbose):
-        _abort("Specify at most one of %s, %s" % (_switch(STR.quiet), _switch(STR.verbose)))
+def _check_verbosity(args) -> Args:
+    if args.get(STR.quiet) and (args.get(STR.debug) or args.get(STR.verbose)):
+        _abort(
+            "Specify at most one of %s, %s, or %s"
+            % (_switch(STR.debug), _switch(STR.quiet), _switch(STR.verbose))
+        )
```

Comment:
I think the abort message could be a little more nuanced, as it's ok (if unnecessary) to use `--verbose` and `--debug` together, and the message says it isn't. Would it be more accurate if it rendered to something like
```
--quiet may not be used with --debug or --verbose
```

? I think that's more what the boolean conditional above actually says.